test/jobs/inat_import_job_test_doubles.rb

```diff
@@ -0,0 +1,165 @@
+# frozen_string_literal: true
```

Comment:
Is `test/jobs/inat_import_job_test_doubles.rb` for the doubles?
Is there a better way to split them off from the test methods?